src/js/auth/ManagementClient.ts

```diff
@@ -11,25 +11,31 @@ export const auth0ManagementClient = new Auth0MgmtClient({
   domain: issuer.replace('https://', ''),
   clientId: mgmtClientId,
   clientSecret: mgmtClientSecret,
-  scope: 'read:users update:users create:users'
+  scope: 'read:users update:users create:users read:roles'
```

Comment:
See my note in the PR write-up. Very important we change the client scope during deployment.

Reply:
can you post a screenshot of Auth0 test tenant where you added the role? (Auth0 dashboard is so complex. I want to make sure I'm making the same change in prod).

Reply:
<img width="1226" alt="image" src="https://user-images.githubusercontent.com/3641356/235995775-c260e455-94df-4e75-9a84-ebef813fc015.png">